protocol/contracts/leaser/src/contract.rs

```diff
@@ -229,8 +240,15 @@ fn protocols_registry_load(storage: &dyn Storage) -> ContractResult<Addr> {
     Config::load(storage).map(|cfg| cfg.protocols_registry)
 }
 
+fn new_code<C>(new_code_id: C, querier: QuerierWrapper<'_>) -> ContractResult<Code>
+where
+    C: Into<CodeId>,
+{
+    Code::try_new(new_code_id.into(), &querier).map_err(Into::into)
+}
+
 fn migrate_msg() -> impl Fn(Addr) -> LeaseMigrateMsg {
-    move |_customer| LeaseMigrateMsg {}
+    |_customer| LeaseMigrateMsg {}
 }
```

Comment:
I feel like we might have went a bit too far with this. 😂
If not inline the lambda, at least lets make this function just drop the argument /use `_: Addr`/ and return the message instead of a lambda.

Reply:
A lambda has been used since we needed the migration message to be customized. I'd prefer to keep it as such to make further migrations easier.

Reply:
Unless the lambda needs to capture context, becoming a closure, it is unnecessary and way less readable to do it this way.